imitation/util/rollout.py

```diff
@@ -297,8 +297,52 @@ def flatten_trajectories(trajectories: Sequence[Dict[str, np.ndarray]]
       cat_parts["rew"]
 
 
-def generate_multiple(policies, env, n_timesteps, *, truncate=True):
-  """Generate obs-act-obs triples from several policies.
+def generate(policy, env, *, n_timesteps=None, n_episodes=None, truncate=True,
```

Comment:
Maybe call this `generate_transitions` to make it clear that they're undistinguished transitions from a bunch of underlying environment copies?

Reply:
Sounds good to me. `generate_multiple` will become `generate_transitions_multiple`